src/accounts/context/userAccount.tsx

```diff
@@ -73,7 +94,24 @@ export const UserAccountProvider: FC<Props> = React.memo(({children}) => {
     } catch (error) {
       event('multiAccount.retrieveAccounts.error', {error})
     }
-  }, [dispatch])
+  }, [dispatch, defaultAccountId])
```

Comment:
does this satisfy the `// todo` comment above this component? if so, can you remove it please?

Reply:
it does, will remove